serve/tests/unittest/test_sampler.py

```diff
@@ -99,39 +99,39 @@ def _test_logit_bias():
     sampling_params = [
         SamplingParams(logit_bias=logit_bias) for logit_bias in list_logit_bias
     ]
-    sampling_metadata = get_sampling_metadata(sampling_params)
+    sampling_state = get_sampling_state(sampling_params)
 
     expected = torch.clone(logits)
     for batch_size in range(batch_size):
         logit_bias = list_logit_bias[batch_size]
         for idx, val in logit_bias.items():
             expected[batch_size][idx - 1] += val
-    new_logits = adjust_logits(logits, sampling_metadata, vocab_size)
+    new_logits = adjust_logits(logits, sampling_state, vocab_size)
     assert torch.allclose(expected, new_logits)
 
 
 def _test_penalties_checker():
```

Comment:
Can we also add the testcase?

Reply:
My colleague extends tests for sampling parameters, particularly for `repetition_penalty`. See #200

Reply:
It is okay to be very basic, so please add one. We've not been doing this so far but I'd like to recommend every PR to have unittests that validates the very basic functionality at least. With SLM migration, we will install the CI as well.